runtime/opt/taupage/init.d/06-update-sysctl.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+
+import logging
+import sys
+import subprocess
+
+from taupage import configure_logging, get_config
+
+SYSCTL_WHITELIST = ['vm.dirty_background_ratio', 'vm.dirty_ratio', 'vm.overcommit_memory', 'vm_overcommit_ratio',
+                    'vm.swappiness']
+CUSTOM_SYSCTL_CONF = '/etc/sysctl.d/99-custom.conf'
+
+
+def write_file(path, content):
```

Comment:
I mean this function is only called once, we could as well move these two lines of code to `main()`.